package.json

```diff
@@ -1288,6 +1293,21 @@
           "type": "integer",
           "default": 1,
           "description": "Number of header rows to assume. Set to 0 to disable.`"
+        },
```

Comment:
Allow disabling tikzpreview in config.

Reply:
Ahhh. Should definitely have that! :stuck_out_tongue: Added in 32980a0d4c829d197bed989de40929d6ca22b2dc